app/src/main/java/com/supertokens/session/Constants.java

```diff
@@ -8,4 +8,9 @@ class Constants {
     static final String ID_TOKEN_HEADER_KEY = "id-refresh-token";
```

Comment:
this constant should not be required anymore?

Reply:
Please search the whole code base for "id-refresh-token", "idrefreshtoken" and make sure it's not there anywhere (unless it's intentionally needed there) - make sure this is the case for all mobile SDKs